ci/mysql57.sh

```diff
@@ -7,6 +7,7 @@ rm -fr /etc/mysql
 rm -fr /var/lib/mysql
 apt-key add support/5072E1F5.asc # old signing key
 apt-key add support/3A79BD29.asc # 5.7.37 and higher
+apt-key adv --keyserver keyserver.ubuntu.com --recv-keys B7B3B788A8D3785C
```

Comment:
For the other keys, I checked the public key into the repo - this may be archaic knowledge, but at least a few times I recall CI failures because the old keys didn't fetch.